astropy/visualization/units.py

```diff
@@ -0,0 +1,96 @@
+# -*- coding: utf-8 -*-
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+
+from __future__ import (absolute_import, division, print_function,
+                        unicode_literals)
+
+
+__doctest_skip__ = ['quantity_support']
+
+
+import numpy as np
+
+
+def quantity_support():
```

Comment:
I realise this is probably best used as a context manager, but should one perhaps give an argument that allows quantity support to be turned off?